src/v2/Apps/Show/components/ShowMeta.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react"
+import { Link, Meta, Title } from "react-head"
+import { createFragmentContainer, graphql } from "react-relay"
+import { ShowMeta_show } from "v2/__generated__/ShowMeta_show.graphql"
+import { getENV } from "v2/Utils/getENV"
+
+interface ShowMetaProps {
+  show: ShowMeta_show
+}
+
+const ShowMeta: React.FC<ShowMetaProps> = ({
+  show: { name, slug, metaDescription, metaImage },
+}) => {
+  const title = `${name} | Artsy`
+  const href = `${getENV("APP_URL")}/show/${slug}`
```

Comment:
Confirming that `/show/` here is deliberate even though this page lives at `/show2/` for now

Reply:
Yeah, this was intentional since we'll never launch it with show2, though I can change it to show2 if that's less confusing for now.

Reply:
Naw, this is good